### ISSUE TITLE
Figure out why "Name and description" proposed by use_github() are wonky

### ISSUE THREAD

**Issue by jennybc** [r-lib/usethis#569]
I'm in a fresh project made with `create_project()` (from an existing directory). Then called `use_git()`. Then `use_github()`.

```
> use_github()
✔ Setting active project to '/Users/jenny/rrr/wtf-workshop-assets/activities/explore-libraries'
● Check title and description
  Name:        explore-libraries,
  Description: 
Are title and description ok?

1: Yeah
2: Nope
3: Not now

Selection: 
```

What's up with the comma at the end of the proposed name? Is it intended that description be blank?

**Comment by mkearney** [comment]
DIBS

**Comment by jennybc** [comment]
Closed in e23a745b1556be1a08fd142f65d18d2f7d276577
